diff --git a/libero/libero/benchmark/__init__.py b/libero/libero/benchmark/__init__.py
index 2b833d0..5734edb 100644
--- a/libero/libero/benchmark/__init__.py
+++ b/libero/libero/benchmark/__init__.py
@@ -161,7 +161,7 @@ class Benchmark(abc.ABC):
             self.tasks[i].problem_folder,
             self.tasks[i].init_states_file,
         )
-        init_states = torch.load(init_states_path)
+        init_states = torch.load(init_states_path, weights_only=False)
         return init_states
 
     def set_task_embs(self, task_embs):
